nvm.sh

```diff
@@ -1870,6 +1870,7 @@ nvm() {
           echo "$NVM_DIR/*/lib/node_modules removed from \$NODE_PATH"
         fi
       fi
+      unset NVM_BIN NVM_PATH
```

Comment:
should this have `-f` so that it will try to unset it even if it happens not to be set?